packages/bitcore-wallet-client/lib/credentials.js

```diff
@@ -56,16 +57,23 @@ function _checkNetwork(network) {
   if (!_.includes(['livenet', 'testnet'], network)) throw new Error('Invalid network');
 };
 
-Credentials.create = function(coin, network) {
+Credentials.create = function(coin, network, account) {
   _checkCoin(coin);
   _checkNetwork(network);
 
+  if (account && !_.isNumber(account) )
+    throw new Error('Invalid account');
+
+  account = account   || 0;
+
   var x = new Credentials();
 
   x.coin = coin;
   x.network = network;
+  x.account = account;
   x.xPrivKey = (new Bitcore.HDPrivateKey(network)).toString();
   x.compliantDerivation = true;
+  x.use145forBCH = true;
```

Comment:
mixing conditional attributes - it other places using `use0forBCH`, here `use145forBCH`, would probably be clearer to default to 145 and have `useLegacyPurpose`, etc, so that we are defaulting and overriding, instead of setting as option both ways.

Reply:
thanks! updated to `useLegacyCoinType`